packages/mock-account-provider/app/routes/prices.ts

```diff
@@ -1,6 +1,7 @@
 import type { LoaderArgs } from '@remix-run/node'
 import { json } from '@remix-run/node'
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 export function loader({ request }: LoaderArgs) {
```

Comment:
```suggestion
export function loader(_args: LoaderArgs) {
```

Reply:
This complains with 
`Property '_args' does not exist on type 'DataFunctionArgs'.ts(2339)`. Same for `_request`, hence the eslint-disable.